app/assets/javascripts/activeadmin_addons/inputs/nested-select.js

```diff
@@ -150,11 +150,14 @@ var initializer = function() {
       }
 
       if (!!parent) {
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
```

Comment:
Expected indentation of 10 spaces but found 14  indent<br>Infix operators must be spaced                  space-infix-ops